debug/index.html

```diff
@@ -42,113 +42,116 @@
-					})).appendTo(stats);
-		        }
-
-				var log = $('<table>').appendTo($('#trace')),
-					t0 = msg.events[0].timestamp,
-					t1 = msg.events[msg.events.length - 1].timestamp,
-					total = t1 - t0;
-		        for (var i = 1; i < msg.events.length; i++) {
-
-					$("<tr>").append($("<td>", {
-						'class': 'time',
-						text: (msg.events[i].timestamp - t0) + " ms"
-					})).append($("<td>", {
-						text: msg.events[i].message
-					})).appendTo(log);
-
-					t0 = msg.events[i].timestamp;
-		        }
-
-		        log.append('<tr><th>' + total + 'ms</th><td>total</td></tr>');
-			});
-
-		}
-
-		var count = 0;
-		MapCSS.onImagesLoad = function() {
-			count += 1;
-			if (count >= 2) {
-				map.addLayer(kothic);
-			}
-		};
-
-		MapCSS.preloadSpriteImage("osmosnimki", "http://osmosnimki.ru/leaf/icons/osmosnimki.png");
-		MapCSS.preloadSpriteImage("mapnik", "styles/mapnik.png");
-
-		$("#renderer").change(function () {
-			switch($("#renderer").val()) {
-				case 'osmosnimki':
-					map.removeLayer(mapnik);
-					kothic.enableStyle('osmosnimki');
-					kothic.disableStyle('mapnik');
-					break;
-				case 'mapnik':
-					map.removeLayer(mapnik);
-					kothic.enableStyle('mapnik');
-					kothic.disableStyle('osmosnimki');
-					break;
-				case 'tiles':
-					//kothic.disableStyle('mapnik');
-					//kothic.disableStyle('osmosnimki');
-					map.addLayer(mapnik);
-					break;
-			}
-		});
-        $("#overlay").change(function () {
-			if ($("#overlay").attr('checked')) {
-				kothic.enableStyle('surface');
-			} else {
-                kothic.disableStyle('surface');
             }
-        });
 
+            var count = 0;
+            MapCSS.onImagesLoad = function () {
+                count += 1;
+                if (count >= 2) {
+                    console.log("adding layer kothic");
+                    map.addLayer(kothic);
+                }
+            };
+
+            MapCSS.preloadSpriteImage("osmosnimki", "styles/osmosnimki.png");
+            MapCSS.preloadSpriteImage("mapnik", "styles/mapnik.png");
+
+            $("#renderer").change(function () {
+                switch ($("#renderer").val()) {
+                    case 'osmosnimki':
+                        map.removeLayer(mapnik);
+                        kothic.enableStyle('osmosnimki');
+                        kothic.disableStyle('mapnik');
+                        break;
+                    case 'mapnik':
+                        map.removeLayer(mapnik);
+                        kothic.enableStyle('mapnik');
+                        kothic.disableStyle('osmosnimki');
+                        break;
+                    case 'tiles':
+                        //kothic.disableStyle('mapnik');
+                        //kothic.disableStyle('osmosnimki');
+                        map.addLayer(mapnik);
+                        break;
+                }
+            });
+            $("#overlay").change(function () {
+                if ($("#overlay").is(':checked')) {
```

Comment:
I guess this is a browser compatibility thing. Which browsers have been tested? Which ones will not work anymore?

Reply:
Due to the question of @nickw1 I changed this to plain HTML, even tough it is not really part of the Kothic framework.